roles/core/pxe_stack/files/osdeploy/redhat_9.ipxe

```diff
@@ -0,0 +1,38 @@
+#!ipxe
+
+echo |
+echo | Entering osdeploy/redhat_9.ipxe file.
+echo |
+
+echo | Now starting os deployment process.
+echo | RedHat ipxe configuration.
+echo |
+echo | Parameters used:
+echo | > Operating system target: ${eq-distribution} ${eq-distribution-version} ${eq-architecture}
+echo | > Console: ${eq-console}
+echo | > Additional kernel parameters: ${eq-kernel-parameters} ${dedicated-kernel-parameters}
+echo | > Deployment server: ${next-server}
+echo | > Target images: repositories/${eq-repositories-environment}/${eq-distribution}/${eq-distribution-version}/${eq-architecture}/
+echo | > Target repos: repositories/${eq-repositories-environment}/${eq-distribution}/${eq-distribution-version}/${eq-architecture}/
+echo | > Target kickstart: ${eq-equipment-profile}.kickstart.cfg
+echo |
+echo | Loading linux ...
+
+kernel ${images-root}/os/images/pxeboot/vmlinuz initrd=initrd.img inst.stage2=${images-root}/os/ inst.repo=${repository-root}/os/BaseOS/ ks=${kickstart-path} ${eq-console} ${eq-kernel-parameters} ${dedicated-kernel-parameters} ipxe_next_server=${next-server} node_hostname=${hostname}
+
+echo | Loading initial ramdisk ...
+
+initrd ${images-root}/os/images/pxeboot/initrd.img
+
+echo | ALL DONE! We are ready.
+echo | Downloaded images report:
+
+imgstat
+
+echo | Booting in 4s ...
+echo |
+echo +----------------------------------------------------+
+
+sleep 4
+
+boot
```

Comment:
why not use `boot || shell`
it allows you to reset a dhcp + ipxe session if it didn't work the first time.
It's convenient for sys admins 😉

Reply:
Good point, I will try this and implement it in another PR

Reply:
ok thanks